### PR TITLE
az-storage-account: add page

### PR DESCRIPTION
**Pull request by WhileLoop**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/az-storage-account.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# az storage account
+
+> Manage storage accounts in Azure.
+> Part of `azure-cli`.
+> More information: <https://learn.microsoft.com/en-us/cli/azure/storage/account>.
+
+- Create a account in a storage account:
```
Comment (navarroaxel) [inline pages/common/az-storage-account.md:7]: Wait, based on the doc this command is for create an storage account itself :thinking: 
```suggestion
- Create an storage account:
```

File: pages/common/az-storage-account.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`az storage account create --name {{storage_account_name}} --resource-group {{azure_resource_group}} --location {{azure_location}} --sku {{storage_account_sku}}`
+
+- Generate a shared access signature for the account:
```
Comment (navarroaxel) [inline pages/common/az-storage-account.md:11]:
```suggestion
- Generate a shared access signature for a specific storage account:
```

File: pages/common/az-storage-account.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`az storage account generate-sas --account-name {{storage_account_name}} --name {{account_name}} --permissions {{sas_permissions}} --expiry {{expiry_date}} --services {{storage_services}} --resource-types {{resource_types}}`
+
+- List storage accounts:
+
+`az storage account list --resource-group {{azure_resource_group}}`
+
+- Delete a storage account:
```
Comment (navarroaxel) [inline pages/common/az-storage-account.md:19]:
```suggestion
- Delete a specific storage account:
```

**Comment by WhileLoop** [comment]
@navarroaxel Issues fixed.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**